packages/create-remix/templates/fly-stack/app/session.server.ts

```diff
@@ -0,0 +1,20 @@
+import { createCookieSessionStorage } from "remix";
+import invariant from "tiny-invariant";
+
+invariant(process.env.SESSION_SECRET, "SESSION_SECRET must be set");
+
+const sessionStorage = createCookieSessionStorage({
+  cookie: {
+    name: "__session",
+    httpOnly: true,
+    maxAge: 60 * 60 * 24 * 7, // 7 days
+    path: "/",
+    sameSite: "lax",
+    secrets: [process.env.SESSION_SECRET],
+    secure: process.env.NODE_ENV === "production"
+  }
+});
+
+const USER_SESSION_KEY = "userId";
+
```

Comment:
Let's move login/register/logout/createSession stuff in here. Similar to what we do in the jokes example: https://github.com/remix-run/remix/blob/main/examples/jokes/app/utils/session.server.ts

They're going to want to do that eventually anyway so we may as well do that for them.

Reply:
being that I already have a `models/user.ts` file, i added the `createUser` and 
`verifyLogin` function there instead.

Reply:
Ahhh yeah, that's even better 👍